src/app/services/wallet.service.ts

```diff
@@ -311,17 +316,21 @@ export class WalletService {
 
   private calculateTimeSinceLastUpdate() {
     const diffMs: number = this.lastBalancesUpdateTime.getTime() - new Date().getTime();
-    const timeSinceLastUpdate = Math.abs(Math.round((diffMs / 1000 / 60)));
+    const timeSinceLastUpdate = this.convertDecimalToInt(diffMs / 1000);
```

Comment:
`diffMs` and `timeSinceLastUpdate` must be inside `this._ngZone.run(() => {`, otherwise the updating is not done correctly.

Reply:
The comment has been fixed.